chart/templates/jaeger-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+{{ if .Values.opentelemetryOperator.enabled -}}
```

Comment:
This is the same old file like the one [below (shown as deleted)](https://github.com/timescale/tobs/pull/207/files#diff-244f8b88267bbd39adc33961fa72236162f87c283780c2a3c5bb96e7fd45d24f )

With few changes as dropping volumes mounts from configmap to passing them as the args.